ops_test.go

```diff
@@ -57,6 +58,7 @@ func doTestFailure(t *testing.T, cancel bool) {
 	var wg sync.WaitGroup
 	wg.Add(1)
 	op.Go(func() {
+		op.FailIf(gerrors.New("This error should be ignored"))
```

Comment:
Why does this one get ignored? Just curious.

Reply:
Because FailIf only records the last error given to it.

Reply:
Ahhh. D'oh 🤦‍♂️

Thanks!